engine/pom.xml

```diff
@@ -166,7 +166,7 @@
                                     <limit>
                                         <counter>LINE</counter>
                                         <value>COVEREDRATIO</value>
-                                        <minimum>0.50</minimum>
+                                        <minimum>0.02</minimum>
```

Comment:
This does raise the question of how to best get coverage in the engine itself. There's not much in the way of unit tests for individual `Evaluator`s.

Reply:
Because most of the tests uses jackson to convert to json back and forth, I moved them to the jackson branch, which reduces the coverage ratio of the main engine module. It would be nice for organizational purposes if we had the evaluator's mapper in the engine's test cases.